about.qmd

```diff
@@ -0,0 +1,9 @@
+---
+title: "About"
```

Comment:
This looks like a stub file. If it is part of the site it should be updated. If the site does not render it, it should be removed.

Reply:
Confirmed that `about.qmd` is not needed and will be removed